sentinel-extension/sentinel-datasource-zookeeper/src/main/java/com/alibaba/csp/sentinel/datasource/zookeeper/ZookeeperAllDataSource.java

```diff
@@ -0,0 +1,77 @@
+package com.alibaba.csp.sentinel.datasource.zookeeper;
+
+import com.alibaba.csp.sentinel.datasource.Converter;
+import com.alibaba.csp.sentinel.property.SentinelProperty;
+import com.alibaba.csp.sentinel.slots.block.authority.AuthorityRule;
+import com.alibaba.csp.sentinel.slots.block.authority.AuthorityRuleManager;
+import com.alibaba.csp.sentinel.slots.block.degrade.DegradeRule;
+import com.alibaba.csp.sentinel.slots.block.degrade.DegradeRuleManager;
+import com.alibaba.csp.sentinel.slots.block.flow.FlowRule;
+import com.alibaba.csp.sentinel.slots.block.flow.FlowRuleManager;
+import com.alibaba.csp.sentinel.slots.system.SystemRule;
+import com.alibaba.csp.sentinel.slots.system.SystemRuleManager;
+import com.alibaba.csp.sentinel.util.StringUtil;
+import com.google.common.collect.Maps;
+
+import java.lang.reflect.Method;
+import java.util.Map;
+
+/**
+ * wrap all ZookeeperDataSource instance and all register2Property operations
+ *
+ * @author zhaixiaoxiang
+ * @date 2019/4/8
+ */
+public class ZookeeperAllDataSource {
+    private Map<Class, ZookeeperDataSource> clazzDataSourceMap = Maps.newConcurrentMap();
+    private String serverAddr;
+
+    public ZookeeperAllDataSource serverAddr(final String serverAddr) {
+        this.serverAddr = serverAddr;
+        return this;
+    }
+
+    public <W,T> ZookeeperAllDataSource register(String path, Class<W> clazz, Converter<String, T> converter) {
```

Comment:
Since `T` must be `List<W>`,
`public <T> ZookeeperAllDataSource register(String path, Class<T> clazz, Converter<String, List<T> converter)`
is better, to prevement user from using wrong class type.
e.g. 
```java
zookeeperAllDataSource = new ZookeeperAllDataSource()
.serverAddr(remoteAddress)
.register(path1, FlowRule.class, new Converter<String, List<ParamFlowRule>>() {
	@Override
	public List<ParamFlowRule> convert(String source) {
		return JSON.parseObject(source, new TypeReference<List<ParamFlowRule>>() {});
	}
}
```